config/coverage.gradle

```diff
@@ -5,16 +5,19 @@ task jacocoTestReport(type: JacocoReport, dependsOn: ['testDebugUnitTest', 'crea
         csv.enabled false
         html.enabled true
         html.destination "${buildDir}/reports/jacoco/test/html"
-        xml.enabled true
-        xml.destination "${buildDir}/reports/jacoco/test/jacocoTestReport.xml"
+        xml.enabled false
```

Comment:
Could you re-enable the xml report (with the same destination), Coveralls depends on it 😇